cypress/integration/EventForm_spec.js

```diff
@@ -0,0 +1,150 @@
+describe('EventForm', () => {
+
+  beforeEach(() => {
+
+    cy.visit('http://localhost:3000/dashboard')
+      .intercept('http://3d8bf4156a8c.ngrok.io/api/v1/courses', {
+        data: [
+          {
+            id: '1', 
+            type: 'course', 
+            attributes: {
+              city: 'Denver', 
+              cost: '80', 
+              name: 'Green Valley Ranch Golf Club', 
+              phone: '303.371.3131',
+              state: 'Colorado',
+              street: '4900 Himalaya Road', 
+              zip_code: '80249'
+            }
+          }
+        ]
+      })
+      .intercept('http://3d8bf4156a8c.ngrok.io/api/v1/players', {
+        data: [
+          {
+            id: '1', 
+            type: 'players', 
+            attributes: {
+              name: 'Amy', 
+              friends: [2, 3], 
+              events: [2]
+            }
+          }
+        ]
+      })
+      .get('[data-cy = form-link]').click()
+  })
```

Comment:
Nice work on the intercept! 

One thing we might want to do is place this in a command at some point to help de-clutter the integration file.